book/website/collaboration/shared-ownership/shared-ownership-challenges.md

```diff
@@ -0,0 +1,27 @@
+(cl-shared-ownership-challenges)=
+# Challenges with Applying Ownership Models Retroactively
+
+While these project ownership models represent the ideals of open source, there are issues that hinder their universal adoption.
+Specifically, it is difficult to apply shared ownership models retroactively for a few reasons described below.
+
+## Strong Ownership
+
+A “strong-ownership” model is When a single contributor or small group of developers feel a strong sense of ownership over the project.
```

Comment:
change uppercase letter W to lowercase letter w in "When" in A "strong-ownership" model is When a single contributor ...